ns1/README.md

```diff
@@ -1,84 +1,62 @@
-# Agent Check: ns1
+# NS1 + Datadog Integration
 
 ## Overview
 
-This check monitors [ns1][1] through the Datadog Agent.
+This integration monitors NS1 services through the Datadog Agent
```

Comment:
```suggestion
This integration monitors [NS1](https://ns1.com/) services through the Datadog Agent
```

Let's keep the link to NS1 website.

Reply:
```suggestion
This integration monitors [NS1][1] services through the Datadog Agent.
```
Links must be done reference-style, collected at the bottom of the file like before.